tests/e2e/tests/e2e_util/util.py

```diff
@@ -215,11 +227,11 @@ def wait_until_k8s_ready(h: harness.Harness, instances: Union[str, List[str]]):
         hostname = (
             h.exec(instance, ["hostname"], capture_output=True).stdout.decode().strip()
         )
-        result = retry_until_condition(
-            h,
-            instance,
-            ["k8s", "kubectl", "get", "node", hostname, "--no-headers"],
-            condition=lambda p: "Ready" in p.stdout.decode(),
+        result = (
+            stubbornly(retries=15, delay_s=5)
+            .on(h, instance)
+            .until(lambda p: "Ready" in p.stdout.decode())
+            .exec(["k8s", "kubectl", "get", "node", hostname, "--no-headers"])
```

Comment:
@neoaggelos 
the response line here says that the node is `NotReady` -- but  because `Ready` is in there -- so it passes the test.

That's kinda silly.  Was it intentional?  Should we wait for ` Ready` to show up in the line instead?

Reply:
yup, that would be better

Reply:
updated PR and rebased, will merge after CI is green